.coveragerc

```diff
@@ -347,6 +347,7 @@ omit =
     homeassistant/components/message_bird/notify.py
     homeassistant/components/met/weather.py
     homeassistant/components/meteo_france/*
+    homeassistant/components/meteoalarm/sensor.py
```

Comment:
`binary_sensor.py` or maybe even better `*`

Reply:
Ah sorry, my mistake. Also fixes this right now. So hopefully i'm on time for the beta on wednesday.